pkg/graph/schema.resolvers.go

```diff
@@ -1134,6 +1140,15 @@ func (r *mutationResolver) UpdateAccessibilityRequestCedarSystem(ctx context.Con
 	}, err
 }
 
+// CreateSystemIntakeActionProgressToNewStep is the resolver for the createSystemIntakeActionProgressToNewStep field.
+func (r *mutationResolver) CreateSystemIntakeActionProgressToNewStep(ctx context.Context, input *model.SystemIntakeProgressToNewStepsInput) (*model.UpdateSystemIntakePayload, error) {
+	updatedIntake, err := newstep.ProgressIntake(ctx, r.store, r.service.FetchUserInfo, input)
```

Comment:
NIT: I don't think it ultimately matters, but I was curious about this still calling a function in the resolver package, and that function calls the newstep package? 

We'd really only see that the parent level still is calling something in resolvers, and that file imports the newstep package.  I'm not sure that's actually better though.

Reply:
Per our discussion on Tuesday, I moved the top-level `ProgressIntake()` function to `pkg/graph/resolvers/system_intake_actions.go`; this now calls it as `resolvers.ProgressIntake()`.